tests/robot-tests/tests/admin_and_public/bau/release_and_publication_role_permissions.robot

```diff
@@ -0,0 +1,352 @@
+
+    # stale element exception if you don't wait until it's enabled    
+    user waits until button is enabled  Add publication access
+    user scrolls to element  css:[name="selectedPublicationId"]
+    
+    user waits until element is enabled  css:[name="selectedPublicationId"]
+    user selects from list by label  css:[name="selectedPublicationId"]  ${PUBLICATION_NAME}
+
+    user waits until element is enabled  css:[name="selectedPublicationRole"]
+    user selects from list by label  css:[name="selectedPublicationRole"]  Owner
+    user clicks button  Add publication access
+    
+Sign in as Analyst1 User1 (publication owner) & navigate to publication
+    [Tags]  HappyPath
+    user changes to analyst1
+    user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}
+    ...  ${RELEASE_TYPE} (not Live)    
+
+Assert publication owner can upload subject file
+    [Tags]  HappyPath
+    user clicks link  Data and files
+    user waits until page does not contain loading spinner
+    user uploads subject   ${SUBJECT_NAME}  seven_filters.csv  seven_filters.meta.csv
+
+Assert publication owner can add meta guidance to ${SUBJECT_NAME}
+    [Tags]  HappyPath
+    user clicks link  Metadata guidance
+    user waits until page does not contain loading spinner
+    user enters text into element  id:metaGuidanceForm-content  Test meta guidance content
+    user waits until page contains accordion section  ${SUBJECT_NAME}
+    user enters text into meta guidance data file content editor  ${SUBJECT_NAME}
+    ...  meta guidance content
+    user clicks button  Save guidance
+
+Navigate to 'Footnotes' page
+    [Tags]  HappyPath
+    user clicks link  Footnotes
+    user waits until h2 is visible  Footnotes
+
+Assert publication owner can add a footnote to ${SUBJECT_NAME}
+    [Tags]  HappyPath
+    user waits until page contains link   Create footnote
+    user clicks link  Create footnote
+    user waits until page does not contain loading spinner
+    user clicks footnote radio   ${SUBJECT_NAME}   Applies to all data
+    user clicks element  id:footnoteForm-content
+    user enters text into element  id:footnoteForm-content  test footnote from the publication owner! (analyst)
+    user clicks button  Save footnote
+    user waits until h2 is visible  Footnotes
+
+Add public prerelease access list
+    [Tags]  HappyPath
+    user clicks link  Pre-release access
+    user waits until page does not contain loading spinner
+    user creates public prerelease access list   Test public access list
+
+Go to "Sign off" page
+    [Tags]  HappyPath
+    user clicks link   Sign off
+    user waits until page does not contain loading spinner
+    user clicks button  Edit release status
+
+Assert publication owner cannot approve release for immediate publication
+    [Tags]  HappyPath
+    user waits until h2 is visible  Edit release status  30
+    user waits for page to finish loading
```

Comment:
If it is necessary to wait here, I'd change this to `user waits until page to contain element  id:releaseStatusForm-approvalStatus-Approved`